src/Client/Modals/ModalElements.fs

```diff
@@ -0,0 +1,183 @@
+                    ]
+                ]
+            ]
+        ]
+
+    static member Box(title: string, icon: string, content: ReactElement, ?className: string list) =
+        Html.div [
+            prop.className [
+                "rounded shadow p-2 flex flex-col gap-2 border"
+                if className.IsSome then
+                    className.Value |> String.concat " "
+            ]
+            prop.children [
+                Html.h3 [
+                    prop.className "font-semibold gap-2 flex flex-row items-center"
+                    prop.children [
+                        Html.i [prop.className icon]
+                        Html.span title
+                    ]
+                ]
+                content
+            ]
+        ]
+
+    static member BoxWithChildren(children: ReactElement list, ?title: string, ?icon: string, ?className: string list) =
+        Html.div [
+            prop.className [
+                "rounded shadow p-2 flex flex-col gap-2 border"
+                if className.IsSome then
+                    className.Value |> String.concat " "
+            ]
+            prop.children [
+                Html.h3 [
+                    prop.className "font-semibold gap-2 flex flex-row items-center"
+                    if icon.IsSome || title.IsSome then
+                        prop.children [
+                            if icon.IsSome then
+                                Html.i [prop.className icon.Value]
+                            if title.IsSome then
+                                Html.span title.Value
+                        ]
+                    prop.children children
+                ]
+            ]
+        ]
+
+    static member SelectorButton<'a when 'a : equality> (targetselector: 'a, selector: 'a, setSelector: 'a -> unit, ?isDisabled) =
+        Daisy.button.button [
+            join.item
+            if isDisabled.IsSome then
+                prop.disabled isDisabled.Value
+            prop.style [style.flexGrow 1]
+            if (targetselector = selector) then
+                button.primary
+            prop.onClick (fun _ -> setSelector targetselector)
+            prop.text (string targetselector)
+        ]
+
+    static member RadioPluginsBox(boxName, icon, importType: TableJoinOptions, radioData: (TableJoinOptions * string)[], setImportType: TableJoinOptions -> unit) =
+        let myradio(target: TableJoinOptions, txt: string) =
+            let isChecked = importType = target
+            ModalElements.RadioPlugin("importType", txt, isChecked, fun (b: bool) -> if b then setImportType target)
+        ModalElements.Box (boxName, icon, React.fragment [
+            Html.div [
+                for i in 0..radioData.Length-1 do
+                    myradio(radioData.[i])
+            ]
+        ])
+
+    static member checkBox(columns: CompositeColumn [], index, selectionInformation: SelectedColumns, setSelectedColumns: SelectedColumns -> unit) =
```

Comment:
This is way to specialized to be in this namespace and the name is too broad for what it does.

Reply:
fixed